### PR TITLE
Link Index documentation page

### PR DESCRIPTION
**Pull request by Mathieudewet**

### CODE REVIEW

File: src/components/home/Schema.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ const SchemaPart = ({ title, items }) => (
       <h4>{title}</h4>
     </div>
     <div className="schema__group">
-      {items.map(item => <SchemaItem key={item.title} {...item} />)}
```
Comment (dunglas) [inline src/components/home/Schema.js]: Why this change?

File: src/data/menu.js [hunk 2]
```diff
@@ -3,6 +3,10 @@ const menu = [
     text: 'docs',
     rootPath: '/docs',
     submenu: [
+      {
+        text: 'Index',
+        path: '/docs/index',
```
Comment (dunglas) [inline src/data/menu.js]: Can't the patch be just `/docs/`? It would be cleaner.

File: src/components/home/Schema.js (hunk 1 above)
Comment (Mathieudewet) [reply src/components/home/Schema.js]: Because of error can't map in undefined.

Comment (dunglas) [reply src/components/home/Schema.js]: Way is it undefined?

File: src/data/schema.js [hunk 3]
```diff
@@ -1,4 +1,8 @@
 const schema = [
+  {
+    title: 'Index',
+    link: '/docs/index',
```
Comment (dunglas) [inline src/data/schema.js]: Same here

File: src/components/home/Schema.js (hunk 1 above)
Comment (Mathieudewet) [reply src/components/home/Schema.js]: Because of no item in current schema part i believe.
